README.md

```diff
@@ -24,7 +24,7 @@ This app requires a dedicated root domain. **ARM installations are NOT SUPPORTED
 
 ## Configuration
 
-The app's config file is stored in `/opt/yunohost/__APPNAME__/nitter.conf`. 
```

Comment:
This app supports multi instance. If the user installs the app multiple times, the config file could be at `nitter__2`, `nitter__3`, etc.